src/app/shared/components/Breadcrumbs/Containers/BreadcrumbsDriveView.tsx

```diff
@@ -16,12 +17,16 @@ interface BreadcrumbsDriveViewProps {
 const BreadcrumbsDriveView = (props: BreadcrumbsDriveViewProps) => {
   const { namePath } = props;
   const dispatch = useAppDispatch();
+  const { selectedWorkspace } = useAppSelector((state: RootState) => state.workspaces);
 
   const breadcrumbDriveViewItems = (): BreadcrumbItemData[] => {
     const items: BreadcrumbItemData[] = [];
 
     if (namePath.length > 0) {
-      const firstPath = namePath[0];
+      const breadcrumbNamePath = [...namePath];
+      const parsedBreadcrumbNamePath =
+        selectedWorkspace && breadcrumbNamePath.length > 1 ? breadcrumbNamePath.slice(1) : breadcrumbNamePath;
+      const firstPath = parsedBreadcrumbNamePath[0];
```

Comment:
From what I see here you are removing the first element, haven't you found out why?
So it appears with the name Drive in the breadcrumb?

Reply:
The first element when calling the ancestors always corresponds to personal drive, in the case of workspaces I select the second one, which is the root of the workspace.
In this way Drive in the breadcrumbs is always the root folder of drive or workspace depending on what we have selected